test/BUILD.gn

```diff
@@ -1119,6 +1119,7 @@ test("brave_browser_tests") {
       "//brave/browser/brave_resources_browsertest.cc",
       "//brave/browser/ssl/certificate_transparency_browsertest.cc",
       "//brave/browser/ui/views/toolbar/wallet_button_notification_source_browsertest.cc",
+      "//brave/browser/url_sanitizer/url_sanitizer_browsertest.cc",
```

Comment:
put it into a sepratate browser_tests target in b/b/url_sanitizer/BUILD.gn
this old way is kinda deprecated